amp/WEB-INF/src/org/digijava/module/aim/annotations/activityversioning/ActivityVersioningService.java

```diff
@@ -0,0 +1,411 @@
+package org.digijava.module.aim.annotations.activityversioning;
```

Comment:
we can add the code to ActivityVersioningUtil since we already have methods there that does related functionality . We can create a ticket to split that class in order to make it smaller